tensorflow/core/kernels/mkl/mkl_conv_ops.cc

```diff
@@ -1855,25 +1702,28 @@ class MklQuantizedConv2DSumReluOp
       // the scaling factor of 255.0f cancels each other and thus is avoided.
       // If it is not then  it is DT_INT8 and is scaled appropriately.
       if (summand_type == DT_QUINT8)
-        params.post_op_params.push_back(
-            {"sum", ALGORITHM_UNDEF, {scale_summand / scale_output}, ""});
+        params.post_op_params.push_back({"sum",
+                                         mkldnn::algorithm::undef,
+                                         {scale_summand / scale_output},
+                                         ""});
       else
         params.post_op_params.push_back(
             {"sum",
-             ALGORITHM_UNDEF,
+             mkldnn::algorithm::undef,
              {255.0f * scale_summand / (scale_output * 127.0f)},
              ""});
```

Comment:
Nit: Since the command after `if` is multi-line longs, please wrap it with brackets `{}`. Same for `else`.

Reply:
Done

@penpornk Thank you very much for all suggestions!